src/main/scala/org/ergoplatform/nodeView/history/ErgoSyncInfo.scala

```diff
@@ -28,13 +28,13 @@ object ErgoSyncInfo {
 object ErgoSyncInfoSerializer extends Serializer[ErgoSyncInfo] {
 
   override def toBytes(obj: ErgoSyncInfo): Array[Byte] = {
-    scorex.core.utils.concatFixLengthBytes(obj.lastHeaderIds)
+    scorex.core.utils.concatFixLengthBytes(obj.lastHeaderIds.map(idToBytes))
```

Comment:
2x memory spending here

Reply:
`ErgoSyncInfo` message should be quite small + we have plans to rework it in https://github.com/ergoplatform/ergo/issues/125